paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -82,6 +101,15 @@ class PaymentSheet internal constructor(
         DefaultPaymentSheetLauncher(fragment, callback)
     )
 
+    /**
+     * Constructor to be used when launching [PaymentSheet] from a [ComponentActivity] and intending
```

Comment:
```suggestion
     * Constructor to be used when launching [PaymentSheet] from a [Fragment] and intending
```

Should it be like this?

Reply:
Yes, good catch!